src/cli.ts

```
@@ -0,0 +1,156 @@
+    .version('0.1.0'); // Version should ideally be synced with package.json
+
+  program
+    .command('analyze <url>')
+    .description('Analyze a URL to detect fonts and generate optimized font overrides. Optionally provide local font files.')
+    .option('-l, --local-files <files...>', 'Space-separated list of local font files to consider (e.g., path/to/font.ttf path/to/other.woff)')
+    .option('-o, --output-format <format>', 'Output format: "json" or "css" (default: "css")', 'css')
+    .action(async (url: string, options: { localFiles?: string[], outputFormat: 'json' | 'css' }) => {
+      try {
+        // Validate output format explicitly
+        if (!['json', 'css'].includes(options.outputFormat)) {
+          throw new Error(`Invalid output format: "${options.outputFormat}". Must be "json" or "css".`);
```

ESLint

[error] Extra semicolon.

(style/semi)